### PR TITLE
reformatting to python3 using "black -S -l 79"

### PR DESCRIPTION
**Pull request by defaziogiancarlo**
This commit is to separate formatting changes from the actual code changes that will take place when converting to python3.

### CODE REVIEW

File: scripts/llogcolor [hunk 1]
```diff
@@ -30,58 +30,72 @@ import subprocess
...
-
-            'on_black'      :    "\033[40m", 
-            'on_red'        :    "\033[41m",
-            'on_green'      :    "\033[42m",
-            'on_yellow'     :    "\033[43m",
-            'on_blue'       :    "\033[44m",
-            'on_magenta'    :    "\033[45m",
-            'on_cyan'       :    "\033[46m",
-            'on_white'      :    "\033[47m" }
+colors = {
```
Comment (defaziogiancarlo) [inline scripts/llogcolor:33]: The values of `color` could be aligned if anyone cares. The python style guide (PEP 8) says not to for assignment statements, and I don't see anything for dict key-value pairs.

File: scripts/llogcolor [hunk 2]
```diff
@@ -152,9 +190,26 @@ def main():
 
             if options.date:
                 ts = str(datetime.datetime.fromtimestamp(float(ts)))
-                line = thread_color[tid] + ts + ' ' + result.group('begin') + result.group('middle') + tid + result.group('end') + '\n'
+                line = (
+                    thread_color[tid]
+                    + ts
```
Comment (defaziogiancarlo) [inline scripts/llogcolor:195]: The leading `+` here is in the style guide. The rational is that it's more consistent with math notation, although in this case it's concatenation, not arithmetic.

**Comment by ofaaland** [comment]
This is a great start.  I'll review in detail in a bit.  I would like you to modify the commit message, to conform to roughly the following:

1st line: very short description of what changed
2nd line: blank
3rd line/subsequent: describe why you made the change

I know that for this patch, part of the "why" is to make the script conform to our local (LLNL/LC) python formatting standard. Does python3 impose new formatting requirements, that makes this change required for the python3 port?  Or is this patch just to satisfy our internal desire for consistency?

**Comment by defaziogiancarlo** [comment]
ok, better commit message added.

So LC python style guide

**Review by ofaaland — COMMENTED** [review]
Looks good to me, aside from the commit message fixup and the question about alignment for dicts and other such things.

**Pull request closed without merging**